src/pages/Login/Login.js

```diff
@@ -1,6 +1,115 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { BsCheck2, BsArrowRight } from 'react-icons/bs';
 import './Login.scss';
+
 const Login = () => {
-  return <></>;
+  const [userInfoValue, setUserInfoValue] = useState({ email: '', pw: '' });
+  const navigate = useNavigate();
+
+  const onChangeId = event =>
+    setUserInfoValue({ ...userInfoValue, email: event.target.value });
+  const onChangePw = event =>
+    setUserInfoValue({ ...userInfoValue, pw: event.target.value });
```

Comment:
input에 name="email", name="pw"를 주어 계산된 속성명을 사용하여 더 깔끔하게 작성할 수 있을 것 같아요!
```suggestion
  const onChangeUserInfoValue = event =>
    setUserInfoValue({ ...userInfoValue, [event.target.name]: event.target.value });
```

Reply:
감사합니다! 더 깔끔하게 작성할 수 있네요!